parlai/chat_service/core/chat_service_manager.py

```diff
@@ -316,6 +319,12 @@ def _on_new_message(self, message):
             message to put on queue
         """
         agent_id = message['sender']['id']
+        if not self.world_runner.initialized:
```

Comment:
could we abstract this to a function `self.world_runner.is_initialized()`? so that we avoid accessing this prop directly

Reply:
Done